config/initializers/jsonp.rb

```diff
@@ -0,0 +1,80 @@
+require 'rack'
```

Comment:
I'd prefer to use https://github.com/crohr/rack-jsonp instead :) 

If we want to have the code here for some reason, we should copy [their specs](https://github.com/crohr/rack-jsonp/blob/master/spec/rack_jsonp_spec.rb) into our codebase too